src/structures/CommandInteraction.js

```diff
@@ -187,6 +187,22 @@ class CommandInteraction extends Interaction {
    * @property {Role|Object} [role] The resolved role
    */
 
+  /**
+   * Send a followup message to this interaction.
+   * @param {string|APIMessage|MessageAdditions} content The content for the reply
+   * @param {InteractionReplyOptions} [options] Additional options for the reply
+   * @returns {Promise<void>}
```

Comment:
Why not return the created message?

Reply:
I assumed for the same reason none of the other methods do. Will check what it returns.